HHS/Head-Start-TTADP
# [Prod] Download active users - admin

## Description of change
This PR provides a button in the admin UI to enable downloading of all active users. Active users are defined as users that don't have any records in the Permissions table. The downloaded users are in a CSV format.


## How to test
Navigate to the users admin section. Click the "Download users" button. A CSV file "Users.csv" should download.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1414


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
